.rubocop.yml

```diff
@@ -1,27 +1,36 @@
 inherit_from: .rubocop_todo.yml
 
-AllCops:
-  TargetRubyVersion: 2.6
+Gemspec/RequiredRubyVersion:
+  Enabled: false
 
 Layout/EmptyLinesAroundClassBody:
   Enabled: false
 
 Layout/EmptyLinesAroundModuleBody:
   Enabled: false
 
+Layout/LineLength:
+  Exclude:
+    - parse_date.gemspec
+
+Lint/MixedRegexpCaptureTypes:
+  Enabled: false
+
 Metrics/BlockLength:
   Exclude:
     - spec/**/*
 
 Metrics/MethodLength:
   Max: 15
-  ExcludedMethods:
```

Comment:
neat, i didn't even know you could exclude specific methods in the config

Reply:
surprise!